charts/jaeger-operator/templates/crds.yaml

```diff
@@ -1,6 +0,0 @@
-{{- if .Values.crd.install }}
-{{- range $path, $bytes := .Files.Glob "crds/*.yaml" }}
-{{ $.Files.Get $path }}
----
-{{- end }}
-{{- end }}
```

Comment:
This is still useful for systems that use `helm template` to generate manifests but don't use `helm install` to deploy. Particularly, it's useful when upgrading CRDs, since `helm upgrade` doesn't provide support for upgrading CRDs.

I would leave it.